fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -185,7 +188,8 @@ fun AppBar(
                         modifier = Modifier
                             .weight(1F)
                             .padding(token.textPadding(appBarInfo))
-                            .testTag(APP_BAR_SUBTITLE)
+                            .testTag(APP_BAR_SUBTITLE),
+                        horizontalAlignment = if (titleAlignment == TitleAlignment.Center) Alignment.CenterHorizontally else Alignment.Start
```

Comment:
we could directly use Alignment object and skip using if condition

Reply:
removed this